components/applications-service/integration_test/ingestion_test.go

```diff
@@ -604,3 +598,48 @@ func TestIngestHealthUpdatedAtUpdatesOnlyOnHealthUpdate(t *testing.T) {
 		}
 	}
 }
+
+func TestIngestDenySupervisorMemberIDUpdates(t *testing.T) {
+	defer suite.DeleteDataFromStorage()
+
+	var (
+		eventsProcessed = suite.Ingester.EventsProcessed()
+		event           = NewHabitatEvent(
+			withSupervisorId("4f1un3"),
+			withPackageIdent("core/db/0.1.0/20200101121212"),
+			withServiceGroup("db.default"),
+		)
+		bytes, err = proto.Marshal(event)
+	)
+
+	if assert.Nil(t, err) {
```

Comment:
If we use `require.Nil`, or even better, `require.NoError`, we don't need the if then wrapping.

Reply:
Amazing feedback! Thank you, changed already! 💯